core/src/commonMain/kotlin/Descriptor.kt

```diff
@@ -4,13 +4,13 @@ import com.benasher44.uuid.Uuid
 import com.benasher44.uuid.uuidFrom
 
 public fun descriptorOf(
-    serviceUuid: String,
-    descriptorUuid: String,
-    characteristicUuid: String,
+    service: String,
+    characteristic: String,
+    descriptor: String,
```

Comment:
Renamed to not have `Uuid` postfix, as we'll soon support shorthand (e.g. `"heartbeat"`).